src/CftfBundle/Resources/views/DocTree/view.html.twig

```diff
@@ -247,8 +253,14 @@
                                                         data-target="#seeImportLogs" data-alt-document-disabled="true">
                                                     Import Logs
                                                 </button>
-                                            {% endif %}
+                                            {% endif %}                                            
                                         {% endif %}
+                                        {% if 'comments' is active feature %}
```

Comment:
Is there a reason that this `if` block is repeated? (line 202 and 258)

Reply:
One block is for document and other one is for item div. Both div can't see at a time.